hugo.toml

```
@@ -263,6 +263,11 @@ no = 'Sorry to hear that. Please <a href="https://github.com/USERNAME/REPOSITORY
 #	icon = "fa-brands fa-twitter"
 # Replace with fa-x-twitter once available
 
+[[params.links.user]]
+  name_key = "community_bluesky_name"
+  url = "https://bsky.app/profile/kubernetes.bsky.social"
```

The link is to the wrong profile
```suggestion
  url = "https://bsky.app/profile/kubernetes.io"
```

Try this.